packages/modules/imodel-browser/src/containers/iModelGrid/useIModelData.ts

```diff
@@ -77,7 +77,7 @@ export const useIModelData = ({
     }
     const abortController = new AbortController();
 
-    const selection = `?projectId=${projectId}`;
+    const selection = `?iTwinId=${projectId}`;
```

Comment:
Change `project` to `iTwin` i other places including types.

Reply:
hm but then it will  be a breaking change...

Reply:
That move to iTwin was initialy to be intended to be the time where we'd move to 1.0, so maybe that's ok ?

Reply:
Maybe you are right, it is kind of inevitable since current iModel API has changed.